docs/microservices/model/tactical-ddd-content.md

```diff
@@ -15,6 +15,9 @@ This section provides a brief summary of the tactical DDD patterns, so if you ar
 - The attributes of an entity may change over time. For example, a person's name or address might change, but they are still the same person.
 - An entity can hold references to other entities.
 
+> [!NOTE]
+> Be wary of Entity Trap also known as Entity Service Antipattern wherein services are modeled around entities / data instead of behavior. Entity services either expose simple CRUD methods or end up too big to lose their purpose. Service names such as *Manager, *Supervisor, *Handler, *Helper, *Controller can indicate that the associated service is in an entity trap.
```

Comment:
You say "don't model around data" but instead make sure you're modeling around "behavior" -- but then all of your service names are behavior.  See what you can do to reword this to bring more clarity.  Do it WITHOUT using the terms "Entity Trap" and "Entity Service Antipattern" - I feel like those might be attempting to be a shortcut to explaining this.

Just explain the problem using non-jargon.

Also, be sure you include what **harm** would come to your application if you modeled parts of your system this way. Meaning, list the answer to "what's the harm?" -- "too simple or too big" are ambiguous harms and I don't know how that would matter by reading this.

Reply:
Thanks for the comment. Changed it. As per Mark Richards, these are the issues: non-describing names, dumping ground for unrelated functionalities, and overloaded services.

However, I just noticed that in the next page, services have been modelled around entities such as accounts, drone management, etc. It seems to me that author of this article have already fallen for entity trap. What's your advice?

Reply:
I think your rewrite is better, but the inconsistency that is brought forth in this article by it is troubling.  We don't have bandwidth right now to evaluate this situation.  I'm going to close the PR, so we don't have competing guidance in the same article.  On the page, you can provide feedback.  Click the feedback button at the bottom, give the article as a whole a rating and add this observation to the text.  Drop my name in the text as well, that'll be a reminder that we talked.  We review that feedback and backlog items once a week or so.

Reply:
Sure. I will do as you suggested. Thanks